content/en/running/running-with-docker.md

```diff
@@ -1,15 +1,39 @@
 ---
-title: Running with Docker
+title: Running with Docker Desktop
 weight: 50
 type: docs
 ---
 
-As an experiment, we've set up a way to use Medley Interlisp via a pre-built Docker configuration. As part of the experiment,  we've set  up a docker container that contains Xvnc (a kind of protocol gateway) so you would just need a VNC client.
 
-If this is your first time working with Docker, you'll want to [install it](https://docs.docker.com/get-docker/) before continuing.
+### **Running on Windows with Docker Desktop**
 
-You'll also need a modern VNC client; [TightVNC](https://www.tightvnc.com/) works well. Rumor has it that Macs and Linux use VNC normally for screen sharing.
+When running with Docker, Medley runs in a Docker container using the Interlisp/medley image found on Docker Hub.  A VNC Viewer window is used to display the Medley desktop.  All of this is started up using the single `medley` command.
+
+These instructions are for Windows 10 or 11 (x86_64) only.
+
+
+#### To install and run Medley with Docker Desktop:
+
+1.  *Install Docker Desktop:*&nbsp;&nbsp;Install Docker Desktop for Windows as described [here](https://docs.docker.com/desktop/install/windows-install/).
```

Comment:
I'd list this as a prerequisite with a link to Docker's site.

Reply:
As @fghalasz had it, there was a goal of making "one thing to install", and having that installer install all prerequisites. Docker for Desktop and WSL aren't that way now -- if you don't have them you have to install separately.